File: leet_easy_c++/leet_168/leet_168a.cpp
class Solution {
public:
    string convertToTitle(int n) {
        string res = "";
        char buf;
        while(n>0){
            n = n-1;
            buf = 'A' + n%26;
            res = buf + res;
            n/=26;
        }
        return res;
    }
};

File: leet_easy_python/leet_168/leet_168a.py
class Solution(object):
    def convertToTitle(self, n):
        res = ""
        while n>26:
            n-=1
            tmp = n%26
            res = chr(tmp + ord('A')) + res
            n //= 26
        res = chr(n - 1 + ord('A')) + res
        return res
        
        """
        :type n: int
        :rtype: str
        """
        